doc/nrf/config_and_build/board_support/board_names.rst

```diff
@@ -73,6 +73,16 @@ Also see the :ref:`zephyr:boards` section in the Zephyr documentation.
 +-------------------+------------+-------------------------------------------------------------------+---------------------------------------+
 | nRF5340 Audio     | PCA10121   | :ref:`nrf5340_audio_dk_nrf5340 <zephyr:nrf5340_audio_dk_nrf5340>` |  ``nrf5340_audio_dk_nrf5340_cpuapp``  |
 +-------------------+------------+-------------------------------------------------------------------+---------------------------------------+
+| nRF54H20 DK       | PCA10145   | :ref:`nrf54h20pdk_nrf54h20 <zephyr:nrf54h20pdk_nrf54h20>`         | ``nrf54h20pdk_nrf54h20_cpuapp``       |
```

Comment:
Not sure about the placement in the table, is there a specific order the boards should be in?

Reply:
I am also not sure about the order. Maybe @FrancescoSer or @annwoj would have more idea on this.

Reply:
Haven't discussed this really, but from the perspective of this entire table I'd say the placement makes sense. @FrancescoSer do you have some additional information regarding this?

Reply:
As Anna suggested, it appears to be organized by series, and likely within each series, it is ordered by release date. I have no strong preference, we could leave the order as is, unless you have any specific suggestion on this.

Reply:
We "always" listed the series with the lowest numbers at the bottom (highest number comes first). Examples: Product Guide at the bottom of this page: https://www.nordicsemi.com/Products and in the NCS, the list of Working with guides, where nRf91 Series comes at the top.

I would suggest following this "highest number first" order, just for the sake of consistency in all of our docsets. See for example the table in https://docs.nordicsemi.com/bundle/nrf-connect-programmer/page/index.html

Reply:
As Greg suggested, I also agree to follow the above approach for consistency. I had searched for similar example in the other docset, and working with guides for nRF54h and nRF54l follow the above approach.

Reply:
Wasn't aware of this, sounds good!